### ISSUE TITLE
Add actions to navigate manuscripts

### ISSUE THREAD

**Issue by uliska** [wbsoft/frescobaldi#733]
This is just an idea, but it might make navigation faster if there were actions "previous manuscript", "next manuscript" and "open manuscripts dropdown" that could be triggered by keyboard shortcuts. I think if you are working with several manuscripts this may be useful.

**Comment by PeterBjuhr** [comment]
Good idea. But please note that there already is an action for the document chooser.

**Comment by uliska** [comment]
Deferred to next milestone (as this is doesn't seem important enough)
